tests/integration/cqlengine/model/test_model_io.py

```diff
@@ -124,6 +125,101 @@ def test_a_sensical_error_is_raised_if_you_try_to_create_a_table_twice(self):
+            e = columns.Date()
+            f = columns.DateTime()
+            g = columns.Decimal()
+            h = columns.Double()
+            i = columns.Float(double_precision=False)
+            j = columns.Inet()
+            k = columns.Integer()
+            l = columns.Text()
+            m = columns.TimeUUID()
+            n = columns.UUID()
+            o = columns.VarInt()
+
+        sync_table(AllDatatypesModel)
+
+        input = ['ascii', 2 ** 63 - 1, bytearray(b'hello world'), True, date(1970, 1, 1),
+                 datetime.utcfromtimestamp(872835240), Decimal('12.3E+7'), 3.4028234663852886e+38,
+                 0.15625, '123.123.123.123', 2147483647, 'text',
+                 UUID('FE2B4360-28C6-11E2-81C1-0800200C9A66'), UUID('067e6162-3b6f-4ae2-a171-2470b63dff00'),
+                 int(str(2147483647) + '000')]
+
+        AllDatatypesModel.create(id=0, a='ascii', b=2 ** 63 - 1, c=bytearray(b'hello world'), d=True, e=date(1970, 1, 1),
+                                 f=datetime.utcfromtimestamp(872835240), g=Decimal('12.3E+7'), h=3.4028234663852886e+38,
+                                 i=0.15625, j='123.123.123.123', k=2147483647, l='text',
+                                 m=UUID('FE2B4360-28C6-11E2-81C1-0800200C9A66'), n=UUID('067e6162-3b6f-4ae2-a171-2470b63dff00'),
+                                 o=int(str(2147483647) + '000'))
+
+        self.assertEqual(1, AllDatatypesModel.objects.count())
+        output = AllDatatypesModel.objects().first()
+
+        for i, i_char in enumerate(range(ord('a'), ord('a') + 15)):
+            self.assertEqual(input[i], output[chr(i_char)])
+
+    def test_can_insert_double_and_float(self):
+        """
+        Test for inserting single-precision and double-precision values into a Float and Double columns
+
+        test_can_insert_double_and_float tests a Float can only hold a single-precision value, unless
+        "double_precision" attribute is specified as True or is unspecified. This test first tests that an AttributeError
+        is raised when attempting to input a double-precision value into a single-precision Float. It then verifies that
+        Double, Float(double_precision=True) and Float() can hold double-precision values by default. It also verifies that
+        columns.Float(double_precision=False) can hold a single-precision value, and a Double can hold a single-precision value.
+
+        @since 2.6.0
+        @jira_ticket PYTHON-246
+        @expected_result Each floating point column type is able to hold their respective precision values.
+
+        @test_category data_types:primitive
+        """
+
+        class FloatingPointModel(Model):
+            id = columns.Integer(primary_key=True)
+            a = columns.Float(double_precision=False)
+            b = columns.Float(double_precision=True)
+            c = columns.Float()
+            d = columns.Double()
+
+        sync_table(FloatingPointModel)
+
+        with self.assertRaises(AttributeError):
```

Comment:
It looks like this is failing in Travis. Why do you expect an attribute error here? 'a' is a valid attribute, and 'float' is double in Python. I think the only thing we can really validate here is that the DB type is correct. (?)